pydy/system.py

```diff
@@ -610,3 +610,42 @@ def _Kane_constant_symbols(self):
                 *self._Kane_inlist_insyms()))
         constants.remove(dynamicsymbols._t)
         return constants
+
+    def add_gravity(self, direction=None):
```

Comment:
Note that we've represented forces and torques as tuples so far. It may only need to be a namedtuple unless you need some methods for Force and Torque.